File: config/config-sbc2ha-new/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/platform/Mcp23017BusConfig.java
package com.dfi.sbc2ha.config.sbc2ha.definition.platform;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.PlatformType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class Mcp23017BusConfig extends I2cBusConfig {
    public Mcp23017BusConfig() {
        super();
        platform = PlatformType.MCP23017;
    }
}

File: config/editor/src/app/definition/platform/bus/Mcp23017BusConfig.ts
import {PlatformType} from "../../enums/PlatformType";
import {I2cBusConfig} from "./I2cBusConfig";

export class Mcp23017BusConfig extends I2cBusConfig {
    public constructor(busId: string, address: number) {
        super(PlatformType.MCP23017, busId, address);
    }
}
